src/node.cc

```diff
@@ -164,11 +164,16 @@ static const char* icu_data_dir = nullptr;
 // used by C++ modules as well
 bool no_deprecation = false;
 
-#if HAVE_OPENSSL && NODE_FIPS_MODE
+#if HAVE_OPENSSL
+// used by crypto module
+bool openssl_system_conf = false;
+#if NODE_FIPS_MODE
 // used by crypto module
 bool enable_fips_crypto = false;
 bool force_fips_crypto = false;
 #endif
```

Comment:
`# endif`

(I know it is odd, but this is how we usually indent nested defines)